lib/utils/date-format/date-format.test.ts

```diff
@@ -29,7 +29,7 @@ test("is future date", () => {
 });
 
 test("formats the time correctly", () => {
-  expect(formatTime("2021-10-06T11:36:25.805Z")).toBe("11:36:25");
+  expect(formatTime("2023-03-16T11:36:25Z")).toBe("11:36:25");
```

Comment:
Is this for the End date functionality?

Reply:
Before this test was failing and it was going an hour forwards for some reason.

Reply:
What is the Error you get when you run in an IDE?

Reply:
No error, the dates just don't match. It comes up as one day ahead of what it should be.